chain/network/src/network_protocol/mod.rs

```diff
@@ -598,6 +600,10 @@ impl fmt::Debug for RoutedMessageBody {
             RoutedMessageBody::_UnusedVersionedStateResponse => write!(f, "VersionedStateResponse"),
             RoutedMessageBody::ChunkStateWitness(_) => write!(f, "ChunkStateWitness"),
             RoutedMessageBody::ChunkEndorsement(_) => write!(f, "ChunkEndorsement"),
+            // TODO(#10790): Also print the chunk hash.
+            RoutedMessageBody::ChunkStateWitnessAck(ack, ..) => {
+                write!(f, "ChunkStateWitnessAck({})", ack.chunk_hash.0)
```

Comment:
The rest of the code is admittedly doing the wrong thing too, but nevertheless it would be a good idea to use debug builders: https://doc.rust-lang.org/stable/std/fmt/struct.Formatter.html#method.debug_tuple